File: abc_spec/code_samples/C#/disputes@{dispute_id}@evidence/put.cs
// For more information please refer to https://github.com/checkout/checkout-sdk-net
using Checkout.Disputes;

Previous.ICheckoutApi api = CheckoutSdk.Builder()
	.Previous()
	.StaticKeys()
	.PublicKey("public_key")
	.SecretKey("secret_key")
	.Environment(Environment.Sandbox)
	.HttpClientFactory(new DefaultHttpClientFactory())
	.Build();

DisputeEvidenceRequest request = new DisputeEvidenceRequest()
{
	ProofOfDeliveryOrServiceFile = "file_xxxxxx",
	ProofOfDeliveryOrServiceText = "proof of delivery or service text",
	InvoiceOrReceiptFile = "file_xxxxxx",
	InvoiceOrReceiptText = "Copy of the invoice",
	CustomerCommunicationFile = "file_xxxxxx",
	CustomerCommunicationText = "Copy of an email exchange with the cardholder",
	AdditionalEvidenceFile = "file_xxxxxx",
	AdditionalEvidenceText = "Scanned document"
};

try
{
	EmptyResponse response = await api.DisputesClient().PutEvidence("disputes_id", request);
}
catch (CheckoutApiException e)
{
	// API error
	string requestId = e.RequestId;
	var statusCode = e.HttpStatusCode;
	IDictionary<string, object> errorDetails = e.ErrorDetails;
}
catch (CheckoutArgumentException e)
{
	// Bad arguments
}
catch (CheckoutAuthorizationException e)
{
	// Invalid authorization
}

File: abc_spec/code_samples/PHP/disputes@{dispute_id}@evidence/put.php
<?php
//For more information please refer to https://github.com/checkout/checkout-sdk-php

use Checkout\\CheckoutApiException;
use Checkout\\CheckoutArgumentException;
use Checkout\\CheckoutAuthorizationException;
use Checkout\\CheckoutSdk;
use Checkout\\Disputes\\DisputeEvidenceRequest;
use Checkout\\Environment;

$api = CheckoutSdk::builder()
    ->previous()
    ->staticKeys()
    ->environment(Environment::sandbox())
    ->secretKey("secret_key")
    ->build();

$request = new DisputeEvidenceRequest();
$request->proof_of_delivery_or_service_file = "file_id";
$request->proof_of_delivery_or_service_text = "proof of delivery or service text";
$request->invoice_or_receipt_file = "file_id";
$request->invoice_or_receipt_text = "Copy of the invoice";
$request->customer_communication_file = "file_id";
$request->customer_communication_text = "Copy of an email exchange with the cardholder";
$request->additional_evidence_file = "file_id";
$request->additional_evidence_text = "Scanned document";

try {
    $response = $api->getDisputesClient()->putEvidence("dispute_id", $request);
} catch (CheckoutApiException $e) {
    // API error
    $error_details = $e->error_details;
    $http_status_code = isset($e->http_metadata) ? $e->http_metadata->getStatusCode() : null;
} catch (CheckoutAuthorizationException $e) {
    // Bad Invalid authorization
}
